source/reference/minio-mc/mc-replicate-diff.rst

```diff
@@ -0,0 +1,116 @@
+.. _minio-mc-replicate-diff:
+
+=====================
+``mc replicate diff``
+=====================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc replicate diff
+
+Description
+-----------
+
+.. start-mc-replicate-diff-desc
+
+The :mc:`mc replicate diff` shows object versions that are not replicated, either generally for the bucket or to a specific remote bucket.
+
+.. end-mc-replicate-diff-desc
+
+Syntax
+------
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command shows unreplicated objects for the ``teamorange/projects`` prefix in the ``notes`` bucket for the ``myminio`` ALIAS:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc replicate diff myminio/notes/teamorange/projects
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         mc [GLOBALFLAGS] replicate diff   \
+                          [--arn "string"] \
+                          TARGET
+
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: TARGET
+   :required:
+
+   The path to the alias, prefix, or object.
+
+.. mc-cmd:: arn
+   :optional:
+
+   The Amazon Resource Number (ARN) to use for the request.
```

Comment:
```suggestion
   The Amazon Resource Number (ARN) of the remote replication target to use for the diff.
   
   Use :mc-cmd:`mc admin bucket remote ls` to retrieve the configured remote ARNs for the bucket.
```

Reply:
Logic here is that ARN isn't helpful to the user - what we want them to provide is the remote ARN for their preferred target. We have to lead the user a bit here since we diverge from AWS in terms of how/where to get this .

Reply:
I added a graph at the top of the page with a link to the `mc admin bucket remote` heading that explains how to find lists of remote bucket ARNs.